lua/plugins/dap.lua

```diff
@@ -6,8 +6,8 @@ return {
       "jay-babu/mason-nvim-dap.nvim",
       dependencies = { "nvim-dap" },
       cmd = { "DapInstall", "DapUninstall" },
-      opts = { automatic_setup = true },
-      config = require "plugins.configs.mason-nvim-dap",
+      opts = { handlers = {} },
+      config = require "plugins.configs.mason-nvim-dap", -- remove in v4
```

Comment:
Why can't this be removed now?

Reply:
I suppose we can remove the line in our configuration and just leave the file. The main thing is to avoid breaking changes to the user if they are using that file and overriding the config function already. Us removing that file will be a breaking change Technically

Reply:
can we use -- TODO:

Reply:
can keep file and still delete the line